kstring.c

```diff
@@ -153,6 +153,15 @@ int kvsprintf(kstring_t *s, const char *fmt, va_list ap)
 		return l;
 	}
 
+    if (!s->s) {
```

Comment:
This has a different indentation style to the rest of the file.  (kstring.c differs to the rest of htslib as it is imported from elsewhere)

Reply:
That shouldn't matter if you've set your tab size correctly.

Reply:
The entire rest of the file is tab indented, while this change is 4 space indented.  If we're to believe tabs vs space don't matter, then why do we have a very strict regime of absolutely zero tabs (not to mention invisible things like trailing white space) ?

I think it's also a bit much to require people to use text editors with configurable tab settings to adjust tabs to be 4 spaces.  Infact I don't think it's even possible with straight "less".

Reply:
I have replaced the spaces with tabs.